common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -45,7 +45,8 @@
             renderCategoryMap: function(map) {
                 var categoryTemplate = edx.HtmlUtils.template($('#new-post-menu-category-template').html()),
                     entryTemplate = edx.HtmlUtils.template($('#new-post-menu-entry-template').html()),
-                    mappedCategorySnippets = _.map(map.children, function(name) {
+                    mappedCategorySnippets = _.map(map.children, function(nameType) {
```

Comment:
Based on [my other comment](https://github.com/edx/edx-platform/pull/13662/files#r83212620), this change may not be necessary.  However, if we decide it is needed, I think `nameType` should be renamed to `child`, and you could possibly add the comment:
```
// child[0] is the category name, child[1] is the type (i.e. 'entry' or 'subcategory')
```